shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -627,4 +672,32 @@ - (void)scrollWheel:(NSEvent*)event {
   [self dispatchMouseEvent:event phase:kHover];
 }
 
+int MapFile(const char* inPathName, void** outDataPtr, size_t* outDataLength, bool executable) {
+  int outError;
+  int fileDescriptor;
```

Comment:
Variables should be declared as close as possible to their use, and when possible as part of their initialization, per style guide.